santosfamilyfoundation/RuinaMain
# updated date and time

# Description:
Changed date and time in crash and road to a string in the format of YYYY/MM/DD HH:MM instead of a string of just numbers.

# Testing:
1. Run the app
2. Go to crash/road form
3. Check the autofill button and see if the time populates correctly.
